src/assets/locales/en/notification.json

```diff
@@ -49,7 +49,7 @@
       "content": "New user roles have been added for the app <strong>{{app}}</strong> which you can now assign.\n\nNew Roles: {{roles}}"
     },
     "TECHNICAL_USER_CREATION": {
-      "title": "{{appName}} subscription created a new technical user",
+      "title": "{{offer}} subscription created a new technical user",
```

Comment:
I suppose this should to be adjusted in src/assets/locales/de/notification.json as well

Reply:
> I suppose this should to be adjusted in src/assets/locales/de/notification.json as well

@ntruchsess Updated src/assets/locales/de/notification.json, thank you.